### PR TITLE
fix your typo

### PR DESCRIPTION
**Pull request by toutpt**
+include dependencies ;)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -49,6 +49,8 @@ def read(*names):
       zip_safe=False,
       install_requires=[
           'setuptools',
+          'plone.uuid',
+          'plone.app.uuid',
```
Comment (glenfant) [inline setup.py:53]: Do we really need this ? (Thought his was in the stock Plone egg deps). Anyway, I'll merge this ;)

File: src/aws/authrss/browser/configure.zcml [hunk 2]
```diff
@@ -82,11 +82,11 @@
       />
 
   <!-- We override the plone.app.layout.links.viewlets.RSSViewlet -->
-
+  <include package="plone.app.layout"/>
   <browser:viewlet
       name="plone.links.RSS"
       manager="plone.app.layout.viewlets.interfaces.IHtmlHeadLinks"
-      class=".viewlets.RSSViewlet"
```
Comment (glenfant) [inline src/aws/authrss/browser/configure.zcml:89]: I removed (temporarily ?) the viewlet in favour of a monkey patch : overriding a Plone stock viewlet class for a custom layer does not work in Plone 4.1 as it worked in Plone 3.x (didn't try in Plone 4.0).
Dunno if it is a bug or a feature.

File: setup.py (hunk 1 above)
Comment (toutpt) [inline setup.py:53]: I have:
- checkout the package
- bootstrap
- buildout
- instance fg
- create a plonesite
- install the addon : fail ... claim no uuid.

fixed that way

## 

Regards / Cordialement
JeanMichel FRANCOIS

Le 11 févr. 2012 à 12:05, Gilles Lenfant
reply@reply.github.com
a écrit :

File: src/aws/authrss/browser/configure.zcml (hunk 2 above)
Comment (toutpt) [inline src/aws/authrss/browser/configure.zcml:89]: duno but your update is called and with the old zcml you override it
using same class (imported) so ... I don t have
seen any monkey but now your viewlet is override by the classic method
with judt a browser layer

## 

Regards / Cordialement
JeanMichel FRANCOIS

Le 11 févr. 2012 à 12:11, Gilles Lenfant
reply@reply.github.com
a écrit :

**Comment by glenfant** [comment]
Did you test with a Plone 4.0 or a 4.1 ?

**Comment by toutpt** [comment]
With the provided buildout -> Plone 4.0

**Pull request merged**